pandas/tests/base/test_misc.py

```diff
@@ -84,6 +84,16 @@ def test_ndarray_compat_properties(index_or_series_obj):
     assert Series([1]).item() == 1
 
 
+def test_array_wrap_compat():
+    # Note: at time of dask 2022.01.0, this is still used by eg dask
+    # (https://github.com/dask/dask/issues/8580).
+    # This test is a small dummy ensuring coverage
+    orig = Series([1, 2, 3], index=["a", "b", "c"])
+    result = orig.__array_wrap__(np.array([2, 4, 6]))
```

Comment:
is there an actual use case where this gets called organically?

Reply:
I suppose only with old numpy? (not really an idea though) But so in dask's case they are calling it explicitly / manually (like this test does)

Reply:
then wouldn't the reasonable thing to do be either a) see if they really need to call it or b) add a "real" test in test_downstream?

Reply:
> see if they really need to call it

They don't need to, and are already fixing it (but we should still do this PR to not directly break released dask IMO, and I can open an issue to remove it in a year or so, or we can first deprecate it)

> add a "real" test in test_downstream?

I would say that the "real" test is downstream in dask's test suite :) 
In the end I don't really care where the test is put, but I think putting it here without usage of dask is useful (I think the dowstream tests are much easier to miss while developing)

Reply:
Added a downstream dask test, but also kept this one.

Will do a follow-up PR to add a deprecation warning for 1.5

Reply:
Follow-up PR to deprecate it -> https://github.com/pandas-dev/pandas/pull/45517